[stage][162] event-broker: Error: Cannot find module 'uuid'

Fails to start on below, probably https://github.com/mozilla/fxa/pull/4479
```
Error: Cannot find module 'uuid'
Require stack:
- /app/dist/lib/jwts.js
- /app/dist/lib/api/index.js
- /app/dist/lib/proxy-server.js
- /app/dist/bin/worker.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:982:15)
    at Function.Module._load (internal/modules/cjs/loader.js:864:27)
    at Module.require (internal/modules/cjs/loader.js:1044:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.<anonymous> (/app/dist/lib/jwts.js:15:16)
    at Module._compile (internal/modules/cjs/loader.js:1158:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
    at Function.Module._load (internal/modules/cjs/loader.js:901:14)
    at Module.require (internal/modules/cjs/loader.js:1044:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.<anonymous> (/app/dist/lib/api/index.js:9:16)
    at Module._compile (internal/modules/cjs/loader.js:1158:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
    at Function.Module._load (internal/modules/cjs/loader.js:901:14) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/app/dist/lib/jwts.js',
    '/app/dist/lib/api/index.js',
    '/app/dist/lib/proxy-server.js',
    '/app/dist/bin/worker.js'
  ]
```

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-1368)
┆Issue Number: FXA-1368

--- a/packages/fxa-event-broker/package.json
+++ b/packages/fxa-event-broker/package.json
@@ -44,7 +44,8 @@
     "request": "^2.88.2",
     "request-promise-native": "^1.0.8",
     "sqs-consumer": "^5.4.0",
-    "typesafe-joi": "^2.1.0"
+    "typesafe-joi": "^2.1.0",
+    "uuid": "^7.0.2"
   },
   "devDependencies": {
     "@types/chai": "^4.2.10",
@@ -73,7 +74,6 @@
     "tslint-config-prettier": "^1.18.0",
     "tslint-plugin-prettier": "^2.1.0",
     "typesafe-node-firestore": "^1.2.0",
-    "typescript": "^3.8.3",
-    "uuid": "^7.0.2"
+    "typescript": "^3.8.3"
   }
 }

